File: Iniciante/1035 - Teste de Seleção 1/Python/1035.py
def main():
    a, b, c, d = map(int, input().split())
    if are_values_accepted(a, b, c, d):
        print("Valores aceitos")
    else:
        print("Valores nao aceitos")


def are_values_accepted(a, b, c, d):
    return b > c and d > a and (c + d) > (a + b) and c > 0 and d > 0 and a % 2 == 0


if __name__ == "__main__":
    main()

File: Iniciante/1035 - Teste de Seleção 1/TypeScript/1035.ts
import * as fs from "fs";

function main(): void {
  const input: string = fs.readFileSync("/dev/stdin", "utf8");
  const lines: string[] = input.trim().split(/\s+/);
  const [a, b, c, d] = lines.map(Number);
  if (areValuesAccepted(a, b, c, d)) {
    console.log("Valores aceitos");
  } else {
    console.log("Valores nao aceitos");
  }
}

function areValuesAccepted(a: number, b: number, c: number, d: number): boolean {
  return b > c && d > a && c + d > a + b && c > 0 && d > 0 && a % 2 === 0;
}


main();